### DISCUSSION TITLE
Accuracy doesn't show up in progress bar

### DISCUSSION THREAD

**Discussion by lastadreamer** [Lightning-AI/pytorch-lightning#11277] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
```python
class Mynet(pl.LightningModule)
    ... ...
    def training_step(self, batch, batch_idx):
        x, y = batch
        logits = self(x)
        loss = F.nll_loss(logits, y)
        accu = ((torch.argmax(logits, dim=1) == y).sum()/x.shape[0]).item()   
        return {'loss':loss, 'accuracy':accu}
```

i define training_step like this, the progress bar show loss,  but not show accu. 
i want to know why?

pytorch_lightning.__version__ = 1.5.7

**Comment by awaelchli** [comment, marked as answer]
You need to log it: 


```py
self.log("accuracy", accu, prog_bar=True)
return {'loss':loss}
```

Docs: https://pytorch-lightning.readthedocs.io/en/latest/extensions/logging.html#logging-from-a-lightningmodule
